specification/ml/get_buckets/MlGetBucketsRequest.ts

```diff
@@ -24,7 +24,7 @@ import { double, integer } from '@_types/Numeric'
 import { DateTime } from '@_types/Time'
 
 /**
- * Retrieves anomaly detection job results for one or more buckets.
+ * Get anomaly detection job results.
```

Comment:
>  * Get anomaly detection job results.

Should this be "...for buckets" to match the other "... for categories" summary?